eduvpn.spec

```diff
@@ -107,13 +107,13 @@ desktop-file-install %{buildroot}/%{_datadir}/applications/eduvpn-client.desktop
 %doc README.md
 %{_bindir}/letsconnect-cli
 %{_bindir}/letsconnect-gui
-%{_datarootdir}/applications/letsconnect-client.desktop
+%{_datarootdir}/applications/org.letsconnect.client.desktop
```

Comment:
It's called `org.letsconnect-vpn.client` in `variants.py`. Probably a good idea to use the id name everywere.

Reply:
i agree, can you apply that change?

Reply:
You want me to open a new PR to change this one line?

Reply:
i just want it to be fixed. you can also push to this branch. it I snot one line, it is everywhere line and file in this PR where `letsconnect-client` has been changed in `org.letsconnect.client` where it should be `org.letsconnect-vpn.client`

Reply:
done myself.